src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -1,49 +1,70 @@
-<div>
-  <h1 class="color-orange"> Help for Instructors</h1>
-  <div>
-    <p>
-      Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
-      <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
-      <br> If you have any remaining questions, don't hesitate to <a href="mailto:{{ supportEmail }}">email us</a>. We respond within 24 hours.
-    </p>
-    <div id="topics" style="display: block;">
+<div class="main-body" #helpPage>
+  <div id="body">
+    <h1 class="color-orange" id="body"> Help for Instructors</h1>
```

Comment:
Same here.

Reply:
Right this one is unnecessary. I was trying to experiment with scrolls and forgot this one.